src/redux/time-entries/selectors.js

```diff
@@ -1,13 +1,27 @@
 import { createSelector } from 'reselect';
 
+import { clientsItemsSelector } from '../clients/selectors';
+
 export const timeEntriesRootSelector = createSelector(
   state => state,
   ({ timeEntries }) => timeEntries
 );
 
 export const timeEntriesItemsSelector = createSelector(
   timeEntriesRootSelector,
-  ({ items }) => items
+  clientsItemsSelector,
+  ({ items }, clients) =>
+    items
+      .map(timeEntry => ({
+        ...timeEntry,
+        client: clients.find(client => client.id === timeEntry.client)
+          .companyName
+      }))
+      .sort(
+        (timeEntryA, timeEntryB) =>
```

Comment:
For a `.sort()` the arguments `a` and `b` will suffice as that's very recognizable.